lm_eval/tasks/hendrycks_math/math_algebra.yaml

```diff
@@ -0,0 +1,20 @@
+group:
+  - hendrycks_math
+task: math_algebra
+dataset_path: baber/hendrycks_math
+process_docs: !function utils_hendrycks_math.process_docs
+dataset_name: algebra
+output_type: greedy_until
+training_split: train
+test_split: test
+doc_to_text:  !function utils_hendrycks_math.doc_to_text
+process_results: !function utils_hendrycks_math.process_results
+doc_to_target: "{{solution}}"
+generation_kwargs:
+  until:
+    - "$"
+    - "\n"
+  temperature: 0
+  top_p: None
+metric_list:
+  - metric: acc
```

Comment:
Get an `AssertionError` if I add `aggregation` and `higher_is_better` here. Line `550` in `task.py`. Maybe because I'm using `process_results`? It uses the default aggregation and runs normally if I remove them though.

Reply:
`metric` needs to be `exact_match` for `greedy_until`

Reply:
Stupid mistake. Thanks! Fixed in new commit!